src/localization/localization.py

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env python3
+
+import rospy
+
+
+def main():
```

Comment:
this node seems pointless, delete?

Reply:
Leaving it in as a stub in case we need custom localization stuff